examples/plot_simulate_lfp_array.py

```diff
@@ -0,0 +1,106 @@
+"""
+==============
+08. Record LFP
+==============
+
+This example demonstrates how to record local field potentials (LFPs).
+"""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          Mainak Jas <mainakjas@gmail.com>
+#          Christopher Bailey <cjb@cfin.au.dk>
+
+# sphinx_gallery_thumbnail_number = 3
+
+import os.path as op
+
+###############################################################################
+# Let us import ``hnn_core``.
+
+import hnn_core
+from hnn_core import read_params, Network, simulate_dipole
+from hnn_core.parallel_backends import MPIBackend
+
+hnn_core_root = op.dirname(hnn_core.__file__)
+
+###############################################################################
+# Then we read the parameters file
+# params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params_fname = op.join(hnn_core_root, 'param', 'gamma_L5weak_L2weak.json')
+params = read_params(params_fname)
+params['tstop'] = 250
+###############################################################################
+# We will start with simulating the evoked response from
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`.
+# We first instantiate the network.
+# (Note: Setting ``add_drives_from_params=True`` loads a set of predefined
```

Comment:
Is the goal to demonstrate the gamma example? You'll need to indicate in this text box if so since it's referencing the evoked response which isn't implemented any more.

Reply:
How to demonstrate LFP is still up to debate. The example is very much WIP, awaiting the rest of this PR stabilising. I chose gamma because oscillations are more understandable than strong feedforward inputs with lots of spiking.